### PR TITLE
fix checking nvdimm memory target issue

### PR DESCRIPTION
**Pull request by nanli1**
we should check element value one by one
Signed-off-by: nanli <nanli@redhat.com>
```

avocado run --vt-type libvirt --test-runner=runner --vt-machine-type q35 memory.devices.file_nvdimm.lifecycle

 (1/8) type_specific.io-github-autotest-libvirt.memory.devices.file_nvdimm.lifecycle.with_numa.addr_undefined.target_label.access_shared: PASS (227.87 s)
 (2/8) type_specific.io-github-autotest-libvirt.memory.devices.file_nvdimm.lifecycle.with_numa.addr_undefined.target_label.access_private_alignsize: PASS (225.60 s)
 (3/8) type_specific.io-github-autotest-libvirt.memory.devices.file_nvdimm.lifecycle.with_numa.addr_undefined.target_readonly.access_shared: PASS (208.04 s)
 (4/8) type_specific.io-github-autotest-libvirt.memory.devices.file_nvdimm.lifecycle.with_numa.addr_undefined.target_readonly.access_private_alignsize: PASS (195.11 s)
 (5/8) type_specific.io-github-autotest-libvirt.memory.devices.file_nvdimm.lifecycle.with_numa.addr_slot_base.target_label.access_shared: PASS (269.33 s)
 (6/8) type_specific.io-github-autotest-libvirt.memory.devices.file_nvdimm.lifecycle.with_numa.addr_slot_base.target_label.access_private_alignsize: PASS (221.11 s)
 (7/8) type_specific.io-github-autotest-libvirt.memory.devices.file_nvdimm.lifecycle.with_numa.addr_slot_base.target_readonly.access_shared: PASS (210.70 s)
 (8/8) type_specific.io-github-autotest-libvirt.memory.devices.file_nvdimm.lifecycle.with_numa.addr_slot_base.target_readonly.access_private_alignsize: PASS (197.88 s)

```

### CODE REVIEW

**Review by chloerh — APPROVED** [review]

**Review by cliping — APPROVED** [review]

File: libvirt/tests/src/memory/memory_devices/lifecycle_for_file_backed_nvdimm_memory.py [hunk 1]
```diff
@@ -22,6 +22,16 @@
 from provider.memory import memory_base
 
 
+def delete_extra_attr(dict1, dict2):
```
Comment (chunfuwen) [inline libvirt/tests/src/memory/memory_devices/lifecycle_for_file_backed_nvdimm_memory.py:25]: miss param doc, additionally better use meaningful param name ,rather dict1 ,dict2

Comment (nanli1) [reply libvirt/tests/src/memory/memory_devices/lifecycle_for_file_backed_nvdimm_memory.py:25]: hi chunfu, this function is to delete extra attr between any two random dict, do we need to update with more meaningful param name.

Comment (nanli1) [reply libvirt/tests/src/memory/memory_devices/lifecycle_for_file_backed_nvdimm_memory.py:25]: Updated

**Review by chunfuwen — APPROVED** [review]
lgtm

**Pull request merged**